R/derive_joined.R

```diff
@@ -371,7 +375,7 @@ derive_vars_joined <- function(dataset,
     filter_if(filter_add) %>%
     select(
       !!!by_vars,
-      !!!chr2vars(names(order)),
+      !!!replace_values_by_names(order),
```

Comment:
This fixed the duplicates messaging in the reprex in the issue, but to make it work needed to assign the functions calls column names inside the order argument

Reply:
We should not force the users to name the expressions specified for `order`.